src/theme.js

```diff
@@ -1,3 +1,13 @@
 const { default: CreateMuiTheme } = require('@material-ui/core/styles/createMuiTheme');
+const { default: amber } = require('@material-ui/core/colors/amber');
 
-module.exports = CreateMuiTheme({});
+module.exports = CreateMuiTheme({
+    palette: {
+        secondary: {
+            main: amber[500]
```

Comment:
I believe you can just set `secondary: Amber` and that will populate all the secondary colors based on amber, with main being `Amber[500]` as you have set here.

Reply:
More info: https://material-ui.com/customization/palette/

Reply:
Ooh, yeah, that's really cool. I made the change!